packages/block-library/src/heading/block.json

```diff
@@ -1,6 +1,14 @@
 {
 	"name": "core/heading",
 	"category": "common",
+	"selector": {
+		"h1": "h1",
+		"h2": "h2",
+		"h3": "h3",
+		"h4": "h4",
```

Comment:
As referred in https://github.com/WordPress/gutenberg/pull/22698#issuecomment-635301127, we will need some way to know if a given block instance is h4 or not.